packages/geolocation/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "parserOptions": {
+      "ecmaVersion": 2018
+    },
+    "extends": [
+      "eslint:recommended",
+      "prettier"
+    ],
+    "env": {
+      "es6": true,
+      "node": true,
+      "jest": true
+    },
+    "rules": {
+      "no-console": "warn"
+    }
+  }
```

Comment:
@Edilberto-Vazquez Same here related to the end of line

Reply:
I already add the EOL

Reply:
I already add the EOL